src/lib.rs

```diff
@@ -361,19 +405,22 @@ impl Database {
                         return;
                     }
                     Err(err) => {
-                        // Fontconfig parsing failed? Try to load fonts from hardcoded paths then.
+                        // Fontconfig parsing failed?
```

Comment:
Why have you changed this part?

Reply:
Hmm... this was done in case `fontconfig-parser` fails. But I guess it should be fine. Will see.